### DISCUSSION TITLE
Can i set reaction to message?

### DISCUSSION THREAD

**Discussion by tre3p** [rubenlagus/TelegramBots#1072] [Q&A] [answered]
Hello, sorry, i can't find, but i'm interested to react to message with emoji. Something like this:
![photo_2022-05-24 23 54 01](https://user-images.githubusercontent.com/68253357/170130420-029fc671-5cb4-4140-b45b-2df357b08c51.jpeg)

Can i do this with TelegramBots lib?

**Comment by MaicolAntali** [comment]
I don’t think that’s possible. 
Telegram hasn’t made any endpoints available to manage reactions.

**Comment by rubenlagus** [comment, marked as answer]
https://core.telegram.org/bots/api-changelog#december-29-2023
